js/app/restaurant/redux/actions.js

```diff
@@ -1,10 +1,13 @@
-import {createAction} from 'redux-actions'
+import {createAction as createFsAction} from 'redux-actions'
```

Comment:
Would have made less changes to alias the newly added function, instead of renaming the old one. 
Also, what does "FS" mean in `createFsAction`?